index.bs

```diff
@@ -604,6 +613,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of |clientDataJSON|.
                 :   {{AuthenticatorAttestationResponse/attestationObject}}
                 ::  |attestationObject|
+            :   {{ScopedCredential/clientExtensionResults}}
+            ::  A new {{AuthenticationExtensions}} object containing the [=extension identifier=] → [=client extension output=] entries
+                created by running each extension's [=client extension processing=] algorithm to create the [=client extension outputs=],
+                for each [=client extension=] in {{AuthenticatorResponse/clientDataJSON}}.clientExtensions.
```

Comment:
I'd rather handle the (I agree needed) changes to txAuthSimple as a separate PR.  Frankly, I need to review all the actual extensions, but that's not on the top of my queue at the moment.

Elsewhere in the spec it says more about how the client processing works.  I didn't want to repeat it here.  I don't see anyplace that expects to return authenticator extension outputs in client extension outputs.  (There *is* language saying that the client processing rules can *produce* client extension outputs from authenticator extension outputs.)  If there are any ambiguities left, by all means, let me know.